roles/openshift_autoheal/defaults/main.yml

```diff
@@ -3,7 +3,7 @@
 openshift_autoheal_state: present
 
 # The namespace where the components will be created:
-openshift_autoheal_namespace: openshift-autoheal
+openshift_autoheal_namespace: openshift-monitoring
```

Comment:
This makes [openshift_autoheal/tasks/uninstall.yml](https://github.com/ironcladlou/openshift-ansible/blob/monitoring-platform/roles/openshift_autoheal/tasks/uninstall.yml) remove whole openshift-monitoring project, which I think contains other stuff too (openshift_prometheus_operator, openshift_cluster_monitoring_operator)?

In fact roles/openshift_prometheus_operator/tasks/uninstall_prometheus_operator.yaml and roles/openshift_cluster_monitoring_operator/tasks/remove.yaml do the same (hardwired to openshift-monitoring).

Reply:
This was fixed on the PR that just merged to master https://github.com/openshift/openshift-ansible/pull/7753/files#diff-b981b664761bf4f4a8089f29ee799398R5